lib/experimental/disable-tinymce.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * Experiment to disable TinyMCE and the Classic block.
+ *
+ * @package gutenberg
+ */
+
+// add_action( 'admin_footer', 'gutenberg_test_tinymce_access' ); // Uncomment the following line to force an external TinyMCE usage.
+
+/**
+ * Render a variable that we'll use to declare that the editor will need the classic block.
+ */
+function gutenberg_declare_classic_block_necessary() {
+	if ( ! gutenberg_current_content_contains_classic_block() ) {
+		return;
+	}
+	echo '<script type="text/javascript">window.wp.needsClassicBlock = true;</script>';
+}
+add_action( 'admin_footer', 'gutenberg_declare_classic_block_necessary' );
```

Comment:
Can't we use an "editor" specific hook here like block_assets or thing like that?